bin/package-requires-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var packageJson = require( '../' + process.argv[2] );
+var semver = require( 'semver' );
+
+if ( packageJson.private ) {
+	process.exitCode = 0;
```

Comment:
The linter prohibits me from using `process.exit(0)`.  That may be a rule to revisit as it seems like a better approach.